mslib/mscolab/file_manager.py

```diff
@@ -250,6 +254,42 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def save_user_profile_image(self, user_id, image_file):
+        """
+        Save the user's profile image path to the database.
+        """
+        upload_folder = mscolab_settings.UPLOAD_FOLDER
+        file_token = secrets.token_urlsafe(20)
+
+        with fs.open_fs('/') as home_fs:
```

Comment:
Opening "/" seems like a bad idea, potentially being vulnerable to path traversals (we already had something like that this year). Can this be a more specific directory from the get-go?